### PR TITLE
Add heartbeat and lbheartbeat endpoints.

### LINKED ISSUE #34: Add __lbheartbeat__ and __heartbeat__

### PR DESCRIPTION
**Pull request by Natim**
Fixes #34

### CODE REVIEW

File: pollbot/middlewares.py [hunk 1]
```diff
@@ -47,6 +47,7 @@ def error_pages(overrides):
 
 
 async def handle_500(request, response=None, error=None):
+    print(error)
```
Comment (magopian) [inline pollbot/middlewares.py]: did you mean to have this?

File: pollbot/tasks/product_details.py [hunk 2]
```diff
@@ -12,3 +12,12 @@
...
             return '{}-{}'.format(product, version) in body['releases']
+
+
+async def heartbeat():
+    with get_session() as session:
+        url = 'https://product-details.mozilla.org/1.0/firefox.json'
+        async with session.get(url, timeout=10) as resp:
+            if resp.status == 200:
+                return True
+        return False
```
Comment (magopian) [inline pollbot/tasks/product_details.py]: I see duplicated code 😿

File: pollbot/views/utilities.py [hunk 3]
```diff
@@ -18,3 +22,19 @@ def render_yaml_file(filename):
...
+async def heartbeat(request):
+    info = await asyncio.gather(archives.heartbeat(),
+                                bedrock.heartbeat(),
+                                product_details.heartbeat())
+    status = all(info) and 200 or 503
+    return web.json_response({
+        "archive": info[0],
+        "bedrock": info[1],
+        "product-details": info[2]
+    }, status=status)
```
Comment (magopian) [inline pollbot/views/utilities.py]: This indentation seems weird

File: tests/test_tasks.py [hunk 4]
```diff
@@ -134,3 +135,23 @@ class DeliveryTasksTest(asynctest.TestCase):
...
+        # Product Details
+        url = 'https://product-details.mozilla.org/1.0/firefox.json'
+        self.mocked.get(url, status=404)
+
+        resp = await heartbeat(None)
+        assert json.loads(resp.body.decode()) == {
+            "archive": False,
+            "bedrock": False,
+            "product-details": False
+        }
```
Comment (magopian) [inline tests/test_tasks.py]: Do you also want to test the status is 503? (Should it be a 5xx error though? It's not a 5xx from Pollbot but from one of the remote services?)

File: pollbot/middlewares.py (hunk 1 above)
Comment (Natim) [inline pollbot/middlewares.py]: We should probably use a logger here.

**Pull request merged**